docs/howto/#manage-machines.rst#

```diff
@@ -0,0 +1,161 @@
+.. _manage-machines:
```

Comment:
Is this `#manage-machines.rst#` file an editor backup? Presumably it shouldn't be committed.